Makefile

```diff
@@ -4,4 +4,7 @@ clean:
 
 .PHONY: profile
 profile:
-	zip -r -9 profile/webapps/system.gaiamobile.org/application.zip assets bower_components css index.html js manifest.webapp
+	zip -r -9 profile/webapps/system.gaiamobile.org/application.zip assets index.html lib node_modules js manifest.webapp
+
+addon:
+	zip addon.zip addon.js manifest.json assets index.html lib node_modules
```

Comment:
I don't know much about add-ons, but at MozFest we had a session where people were flashing their own add-ons to devices. i don't remember there being a zip stage.